vuestorefront/tests/test_query_website_page.py

```diff
@@ -1,9 +1,12 @@
 from unittest.mock import patch
 
+from odoo.tests.common import HttpCaseCommon, get_db_name
 
 from . import common
 
 PATH_UTILS_REQ = "odoo.addons.vuestorefront.utils.request"
+# TODO: we should redesign rendering to use it directly without external
+# request call. Then we would need to use these paths, so keeping for now.
```

Comment:
As mentioned in the previous comment, it is good to share the full picture of it so we know what are the issues with the previous approach and why it was not implemented.
Otherwise, if I read this for the first time I would not be aware that other approach was implemented but was buggy and kind of difficult to achieve.